modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -39,6 +39,9 @@
 
 	var STR_RIGHT_SQUARE_BRACKET = ']';
 
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
+	 */
```

Comment:
Do we want to deprecate global variable that contains utils? This `Window` is a part of `Liferay.Util.Window.getWindow`.

Reply:
It's a mess, yea. Took me a bit to find [where it was being populated](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util_window.js#L104). 

Removing this deprecation sounds good 👍